src/main/jbake/content/blog/2017/12/running-agent.adoc

```diff
@@ -0,0 +1,155 @@
+= Running Hawkular Agent
+John Mazzitelli
+2017-12-11
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, agent, prometheus, metrics
+
+Now that Hawkular is moving towards integraton with Prometheus as its metrics collection and storage system, the agent
+has had some changes. This document will discuss those changes and how to run the Hawkular Agent.
+
+First, the Hawkular Agent now only runs as a javaagent (no longer does it run inside WildFly as a subsystem extension).
```

Comment:
Do we want to add specifics as to when this change hit?  (As of version X, the Hawkular agent now only runs as a javaagent...)  Especially since this blog post is going to live out there for a while, or folks might find it via Google search.

Reply:
will add a blurb about this being for 2.0.0.Final